cognite/client/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.42.0"
-__api_subversion__ = "V20220125"
+__version__ = "2.43.0"
+__api_subversion__ = "V20220323"
```

Comment:
this one should remain untouched (I don't know when it should be changed, it is undocumented)

Reply:
Nice to know. We definitely need to document this somewhere

Reply:
So please remove the change on `__api_subversion__` and fix the conflicts to get approval @tuanng-cognite :)

Reply:
Documentation has been requested earlier. Hasn't happened yet.